src/main/java/school/lemon/changerequest/java/introduction/hw1/Task6.java

```diff
@@ -1,11 +1,28 @@
 package school.lemon.changerequest.java.introduction.hw1;
 
-public class Task6 {
+public class Task6  {
+
     public static double calculateS(double x) {
-        return 0d;
+        double s;
+        s = 1 + x + Math.pow(x, 2) / fact(2) + Math.pow(x, 3) / fact(3) + Math.pow(x, 4) / fact(4);
```

Comment:
You can do this calculations in a loop. So if you'll need do it 10 times for example it would be easier to scale it.